deploy-disconnected-registry/common.sh

```diff
@@ -130,8 +157,12 @@ if [[ ${1} == "hub" ]]; then
     export OPENSHIFT_RELEASE_IMAGE="quay.io/openshift-release-dev/ocp-release:${OC_OCP_TAG}"
     export SOURCE_REGISTRY="quay.io"
     export SOURCE_INDEX="registry.redhat.io/redhat/redhat-operator-index:v${OC_OCP_VERSION_MIN}"
+    if [[ ${CUSTOM_REGISTRY} == "false" ]]; then
+        LOCAL_REG="$(oc get route -n ${REGISTRY} ${REGISTRY} -o jsonpath={'.status.ingress[0].host'})"
+    fi
```

Comment:
if custom_registry == true, the LOCAL_REG should contain LOCAL_REG=${EDGECLUSTERS_REGISTRY}, right????